src/createServer.js

```diff
@@ -1,8 +1,18 @@
 'use strict';
 
-const createServer = () => {
-  // your code goes here
-};
+const express = require('express');
+
+const { router: usersRouter } = require('./routes/user.route');
+const { router: expensesRouter } = require('./routes/expenses.route');
+
+function createServer() {
+  const app = express();
+
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
```suggestion
  app.use(express.json());
  
  app.use('/users', usersRouter);
  app.use('/expenses', expensesRouter);
```
You can apply to all routes at once